packages/components/src/components/InfoItem/utils.ts

```diff
@@ -0,0 +1,64 @@
+import { spacings, SpacingValues, TypographyStyle } from '@trezor/theme';
+
+import { FlexAlignItems } from '../Flex/Flex';
+import { IconSize } from '../Icon/Icon';
+import { InfoItemVerticalAlignment } from './types';
+
+export const mapVerticalAlignmentToAlignItems = (
+    verticalAlignment: InfoItemVerticalAlignment,
+): FlexAlignItems => {
+    const alignItemsMap: Record<InfoItemVerticalAlignment, FlexAlignItems> = {
+        top: 'flex-start',
+        center: 'center',
+        bottom: 'flex-end',
```

Comment:
shouldn't we keep `flex-start`, `flex-end` since we use this in Flex components? Or the opposite - shouldn't we unify it to this solution in Flex?

Reply:
I don't think it's the same padding. We might, for example, decide that the "top" in the context of InfoItem is not flex-start, but baseline (it was like that before, I changed it because it caused problems in some cases).